src/styles/index.scss

```diff
@@ -1,3 +1,141 @@
 body {
   margin: 0;
+  display: block;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.card {
+  box-sizing: border-box;
+  min-height: 406px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    flex-shrink: 0;
+    margin: 0 auto 40px;
+  }
+
+  &__body {
+    display: block;
+    // padding: 16px;
+    // background-color: #5e1414;
+  }
+
+  &__title {
+    color: var(--Main-accent, #060b35);
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px; /* 150% */
+    margin: 0;
+  }
+
+  &__product-code {
+    color: var(--Secondary, #616070);
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px; /* 140% */
+    margin-top: 4px;
+  }
+
+  &__stars-rewiews {
+    margin-top: 16px;
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__rewiews {
+    display: flex;
+    width: 53px;
+    height: 16px;
+    flex-direction: column;
+    justify-content: flex-end;
+    flex-shrink: 0;
+    color: var(--Main-accent, #060b35);
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px; /* 140% */
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__price-name {
+    color: var(--Secondary, #616070);
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px; /* 150% */
+  }
+
+  &__price-value {
+    color: var(--Main-accent, #060b35);
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px; /* 112.5% */
+  }
+
+  &__btn {
+    margin-top: 16px;
+    padding: 0;
+    border: 0;
+    width: 166px;
+    height: 40px;
+    flex-shrink: 0;
+    border-radius: 5px;
+    background: var(--Blue-accent, #00acdc);
+    color: var(--White, #fff);
+    text-align: center;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px; /* 114.286% */
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &__btn:hover {
+    background: #fff;
+    color: var(--Blue-accent, #00acdc);
+    border: 1px solid #00acdc;
```

Comment:
don't add border on hover, add transparent one on default state and it's color on hover